app/models/container_node.rb

```diff
@@ -79,4 +79,18 @@ def ipaddress
   def cockpit_url
     URI::HTTP.build(:host => ipaddress, :port => 9090)
   end
+
+  def tenant_identity
+    if ext_management_system
+      ext_management_system.tenant_identity
+    else
+      User.super_admin.tap { |u| u.current_group = Tenant.root_tenant.default_miq_group }
+    end
+  end
+
+  def evaluate_alert(_alert_id, _event)
+    # currently only EmsEvents from hawkular are tested for node alerts,
+    # and these should automaticaly be translated to alerts.
+    true
```

Comment:
Fine for now, because we don't currently support regular MiqExpression-evaluating alerts on container entities.
But would it be hard to check something on the event to confirm it's a hawkular alerting event, just in case?